### PR TITLE
Add Bella Pequette to ARFC

### PR DESCRIPTION
**Pull request by bellapequette**
## Summary of changes
Add Bella Pequette to the Current people page of the ARFC website 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #

## Associated Developers

- Dev: @

## Checklist for Reviewers

Reviewers should use [this link](https://arfc.github.io/manual/guides/pull_requests) to get to the
Review Checklist before they begin their review.

### CODE REVIEW

**Review by nsryan2 — APPROVED** [review]
These changes look good to me (LTGM)! Welcome to the group. I'm working on building on my end to double check, which is why I haven't merged yet. Once that happens, or if someone builds it first, then we'll merge.

File: Gemfile [hunk 1]
```diff
@@ -1,14 +1,14 @@
 source 'https://rubygems.org'
 
-gem 'jekyll', '~> 3.9.2'
-gem 'kramdown', '~> 2.3.2'
```
Comment (katyhuff) [inline Gemfile:4]: @bellapequette were you unable to build with this tilde in place?

**Review by katyhuff — CHANGES_REQUESTED** [review]
These changes to the Gemfile are my only concern -- I assume they were required locally, but they might break our github actions situation...

File: Gemfile (hunk 1 above)
Comment (bellapequette) [reply Gemfile:4]: @nglaser3 identified an issue with the "Jekyll serve" command and helped me resolve it by removing the tilde. It was unable to be built with the tilde in place.

**Comment by nsryan2** [comment]
> These changes to the Gemfile are my only concern -- I assume they were required locally, but they might break our github actions situation...  

I totally forgot to update this thread! I went down a long rabbit hole that basically resulted in that. To build locally the changes worked if I had the newest version of Jekyll, but removing the GitHub actions gem prevented the local version of GitHub actions I ran from working, and it didn't like the other changes to the Gemfile.

@bellapequette if you could change the gemfile on your fork back we can move forward with merging. Happy to discuss this with you if you have questions.

**Pull request merged**